test/end-to-end/shared/GridUtils.js

```diff
@@ -29,6 +29,13 @@ function getCell(gridId, row, col) {
     .element(by.repeater('(colRenderIndex, col) in colContainer.renderedColumns track by col.uid').row(col));
 }
 
+function getCellName(gridId, row, col) {
```

Comment:
You can just remove the old `getCell()` method.  It's obviously for an outdated version of `ui-grid`.